st2tests/st2tests/base.py

```diff
@@ -488,7 +488,7 @@ def assertTriggerDispatched(self, trigger, payload=None, trace_context=None):
 
 class BaseActionTestCase(TestCase):
     """
-    Base class for action tests.
+    Base class for Python runner action tests.
```

Comment:
I will eventually need to rename this to `BasePythonRunnerActionTestCase`, but I will do this in a different PR when adding support for tests for other runners.

I will also leave old name in place for backward compatibility reasons.